.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    "env",
+    "react"
+  ],
+  "plugins": [
+    "transform-class-properties",
+    "transform-object-rest-spread"
+  ]
+}
```

Comment:
you should think about prettier to nicely format the code or at least vs code extension called `add new line`. 
What is wrong with this file? Unix systems require empty line at the end of the each file. This issue is currently everywhere. Each file has to be improved.

Reply:
not all files has been improved. You can review this PR and check where is this sign:
<img width="173" alt="Zrzut ekranu 2019-07-22 o 08 21 36" src="https://user-images.githubusercontent.com/29004768/61610585-d52be000-ac59-11e9-8946-f3253aa19a14.png">